RTN-004.tex

```diff
@@ -244,28 +263,32 @@ \subsection{Equitable Access}\label{ssec:sel_equity}
 A staged application process is implemented to maximize the inclusion of delegates who would not be able to make a time commitment at the start of DP0.1, and because it is recognized that it might take time to achieve a diverse set of DP0 delegates and to provide equitable access to DP0 (Section \ref{ssec:sel_time}).
 
 On-boarding resources and a supportive community will be generated and maintained by the CET (Section \ref{ssec:res_sup}).
-Such resources are recognized as necessary for delegates to acquire the benefits of DP0 participation, regardless of their past experience or expertise. 
+Such resources are recognized as necessary for delegates to acquire the benefits of DP0 participation, regardless of their past experience or expertise.
+
+Flexible time commitments mean that delegates are free to do their DP0-related activities on any timescale which works for them, and that there is no minimum time commitment or expected ``start date" (Section \ref{sssec:intro_del_resp}).
```

Comment:
whatever timescale works for them, and ....

Reply:
fixed